source/screens/caseScreens/CaseOverview.js

```diff
@@ -201,15 +213,21 @@ function CaseOverview(props) {
         const formCases = getCasesByFormIds(formIds);
         const updatedFormCaseObjects = formCases.map(async (caseData) => {
           const form = await getForm(caseData.currentFormId);
-          const component = computeCaseCardComponent(caseData, form, caseType, navigation);
+          const component = computeCaseCardComponent(
+            caseData,
+            personalNumber,
+            form,
+            caseType,
+            navigation
+          );
           return { component, ...caseData };
         });
         return Promise.all(updatedFormCaseObjects).then((updatedItems) => updatedItems);
       });
 
       await Promise.all(updateCaseItemsPromises).then((updatedItems) => {
         const flattenedList = updatedItems.flat();
-        flattenedList.sort((a, b) => (a.updatedAt < b.updatedAt ? 1 : -1));
+        flattenedList.sort((a, b) => b.updatedAt - a.updatedAt);
```

Comment:
The `a` and `b` variabels could be named in a more explicit manner. Since this sort function is not a pure/reusable function :)

Mostley due to that it's not obvious to me when reading this what the flattenedList variable contains :)